src/main/resources/db/migration/v0.37.0.49__flatten_assocations.sql

```diff
@@ -576,4 +581,71 @@ ALTER TABLE ONLY association ADD CONSTRAINT fkeot29m2x4peefkgwrqmu36ub1 FOREIGN
 ALTER TABLE ONLY association ADD CONSTRAINT fkd6l423ab6u2hfoyjaf85nxt4f FOREIGN KEY (aggregationdatabase_id) REFERENCES ontologyterm(id);
 ALTER TABLE ONLY association ADD CONSTRAINT fkkbi83cmvdnis7mdsx966rvdxe FOREIGN KEY (detectionmethod_id) REFERENCES ontologyterm(id);
 ALTER TABLE ONLY association ADD CONSTRAINT fkfk71crh2hl7wuwpmrwvcybnq3 FOREIGN KEY (transcriptcodingsequenceassociationobject_id) REFERENCES codingsequence(id);
+
+ALTER TABLE ONLY genegeneticinteraction_phenotypesortraits ADD CONSTRAINT fklufk1y85ki3qeo1ixbko56rol FOREIGN KEY (genegeneticinteraction_id) REFERENCES association(id);
+
+
+-- Fix sqtr long names;
+
+DROP INDEX sequencetargetingreagent_secondaryIdentifiers_sequencetargetingreagent_index;
+DROP INDEX sequencetargetingreagent_synonyms_sequencetargetingreagent_index;
+
+CREATE INDEX sqtr_secondaryidentifiers_sqtr_index ON sequencetargetingreagent_secondaryidentifiers USING btree (sequencetargetingreagent_id);
+CREATE INDEX sqtr_synonyms_sqtr_index ON sequencetargetingreagent_synonyms USING btree (sequencetargetingreagent_id);
+
+-- Missing Indexes
+
+CREATE INDEX biologicalentity_curie_index ON public.biologicalentity USING btree (curie);
+CREATE INDEX biologicalentity_modinternalid_index ON public.biologicalentity USING btree (modinternalid);
+CREATE INDEX cellularcomponentqualifiers_cellularcomponentqualifiers_index ON public.anatomicalsite_cellularcomponentqualifiers USING btree (cellularcomponentqualifiers_id);
+CREATE INDEX informationcontententity_curie_index ON public.informationcontententity USING btree (curie);
+CREATE INDEX ontologyterm_curie_index ON public.ontologyterm USING btree (curie);
+CREATE INDEX ontologyterm_isa_ancestor_descendant_isancestors_index ON public.ontologyterm_isa_ancestor_descendant USING btree (isaancestors_id);
+
+-- Missing Constrains
+
+ALTER TABLE ONLY public.anatomicalsite ADD CONSTRAINT fk4h252bd81x75fwxgyy6wa61u7 FOREIGN KEY (anatomicalstructure_id) REFERENCES public.ontologyterm(id);
```

Comment:
Can we give these constraints non-cryptic names?

Reply:
The thing is that hibernate makes that hash as a combination of the table and field name. Yes we could name them whatever we want, but it would be good to have that name in java. I am not sure where the FK definition goes in java, I would guess where the index is defined.